apps/ledger-live-mobile/src/screens/SendFunds/04-Summary.tsx

```diff
@@ -269,6 +279,15 @@ function SendSummary({ navigation, route }: Props) {
           navigation={navigation}
           route={route}
         />
+
+        {operation && isEditableOperation(account, operation) ? (
```

Comment:
just a question, wouldn't it be true for a normal sending flow the above condition?
because a new operation is an optimistic pending transaction too. So will it impact the normal sending flow? Is this screen used by sending flow

Reply:
the `isEditableOperation` checks if there is the `operation.transactionRaw` attribute. AFAIK it's only present in a editable operation. This won't be true for a normal transaction

Reply:
yes, maybe just
"operation && operation.transactionRaw"
should be easier ?